### ISSUE TITLE
Test for UI a11y

### ISSUE THREAD

**Issue by danielweck** [readium/readium-desktop#87]
https://github.com/reactjs/react-a11y

**Comment by danielweck** [comment]
General pointers:
https://reactjs.org/docs/accessibility.html

https://medium.com/@svinkle/creating-accessible-react-apps-38dc1c2775cf
https://blog.usejournal.com/getting-started-with-web-accessibility-in-react-9e591fdb0d52

**Comment by danielweck** [comment]
Related: https://github.com/edrlab/readium-desktop/issues/88

**Comment by danielweck** [comment]
Spectron and Devtron with a11y support:

https://electronjs.org/docs/all#accessibility
https://electronjs.org/docs/tutorial/accessibility

Spectron:
https://github.com/electron/spectron#accessibility-testing

Devtron:
https://github.com/electron/devtron

**Comment by clebeaupin** [comment]
This issue should be moved to a next major release

**Comment by panaC** [comment]
What do you mean ?

**Comment by danielweck** [comment]
DevTron is integrated as a Web Inspector plugin (tab), next to React and Redux dev tools. DevTron includes the same a11y checker as Spectron (frankly, quite out of date and not very helpful).

I added the Axe React accessibility checker, see PR: https://github.com/readium/readium-desktop/pull/589
